resources/definitions/ultimaker_s6.def.json

```diff
@@ -0,0 +1,53 @@
+{
+    "version": 2,
+    "name": "UltiMaker S6",
+    "inherits": "ultimaker_s8",
```

Comment:
Shouldn't it inherit S5 ?

Reply:
No. We need to inherit the settings from the high speed printer.